datajunction-query/djqs/api/main.py

```diff
@@ -30,10 +34,13 @@
         "url": "https://mit-license.org/",
     },
 )
-app.include_router(catalogs.router)
-app.include_router(engines.router)
+app.include_router(catalogs.get_router)
+app.include_router(engines.get_router)
 app.include_router(queries.router)
 app.include_router(tables.router)
+if settings.enable_dynamic_config:  # pragma: no cover
```

Comment:
Does it make sense to load the config above, if here you find out that `enable_dynamic_config=True` ?

Reply:
I didn't see any reason to prevent a setup where both a configuration file is used for priming the system with a set of catalogs and engines, but then more catalogs and engines can be added dynamically through API calls.